desktop-exporter/sample_data.go

```diff
@@ -0,0 +1,85 @@
+package desktopexporter
+
+import (
+	"context"
+	"encoding/hex"
+	"log"
+	"time"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+func GenerateSampleData(ctx context.Context) []SpanData {
+	traceData := ptrace.NewTraces()
+
+	traceData.ResourceSpans().EnsureCapacity(1)
+
+	// Resource data for currencyservice
+	currencyResourceSpan := traceData.ResourceSpans().AppendEmpty()
+	currencyResource := currencyResourceSpan.Resource()
+	currencyResource.SetDroppedAttributesCount(0)
+	currencyResource.Attributes().PutStr("service.name", "SAMPLE: currencyservice")
+	currencyResource.Attributes().PutStr("telemetry.sdk.language", "cpp")
```

Comment:
all the cool kids use Rust these days I hear

Reply:
Currency conversion isn't cool tho.